tests/task/test_continuation.py

```diff
@@ -358,7 +358,7 @@ def test_continuation(params: TestParams):
     task_labels.sort()
 
     # Retain a copy of the task graph in the data dir to aid in debugging.
-    task_graph_json = (Path(__file__).parent / "../artifacts/task-graph.json").resolve()
+    task_graph_json = Path(__file__).resolve().parents[2] / "artifacts/task-graph.json"
```

Comment:
I think this can be:

```suggestion
    task_graph_json = (Path(__file__).parent / "../../artifacts/task-graph.json").resolve()
```

Or you can add at the top of the file:

```
ROOT_DIR = (Path(__file__).parent / "../..").resolve()
```

And then here:

```
task_graph_json = ROOT_DIR / "artifacts/task-graph.json"
```

Which would simplify this code.